imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -724,6 +731,8 @@ public void verifyDetailFileChecksumAndCount(CdrFileNotificationRequest request)
                 null
         ));
 
+        // TODO: is there a better place/way to trigger this?
+        cleanOldCallRecords();
```

Comment:
kind of like what we do for generateTargetFile (except we don't do a cron scheduler - which we actually should have done in the first place)